src/Components/Publishing/Layouts/NewsLayout.tsx

```diff
@@ -123,7 +123,7 @@ export class NewsLayout extends Component<Props, State> {
     return (
       <>
         {shouldAdRender && (
-          <NewDisplayCanvas
+          <DisplayAd
             pt={is300x50AdUnit(adDimension) ? 2 : 4} // add 20px to mobile leaderboard ads until this component is converted to <DisplayAd />
```

Comment:
I ran into the same issue [as my other similar PR](https://github.com/artsy/reaction/pull/2592#discussion_r300110145) when I removed this line, so I left it for now.

Reply:
Were you able to confirm this behavior happens on a mobile device? I saw the screenshot you posted but, but I have noticed that the Chrome mobile simulator doesn't behave exactly the same as actual mobile devices.

Reply:
If you haven't tried browsing sites served through your local dev server, lmk and we can go through how to get it up. I had to QA some mobile fixes on my phone via my machine running on localhost recently.